### ISSUE TITLE
Syntax highlighting fails in Swift

### ISSUE THREAD

**Issue by jfl913** [kolinkrewinkel/Polychromatic#49]
Xcode 6.2. It becomes white and black. Color dismiss.

**Comment by jtdavid** [comment]
Indeed.

**Comment by ksalak** [comment]
looking forward to get it fixed! already get used to this great plug-in!

**Comment by Paul-van-Klaveren** [comment]
Hi, I am still really enjoying this plugin, but I am experiencing the same problem in Xcode 6.4 (6E35b), using the latest Polychromatic release ( v1.1.1 manually as well as via Alcatraz).

Polychromatic is working great in most of my Swift files, but in one file it stops working in a (rather lengthy) private function. I've reproduced the crash in a fake method:

```swift
private func test(dictionaryOfDictionaries: NSDictionary) {
        if let test = "whatever" ?? (dictionaryOfDictionaries["meta"] as? NSDictionary)?["id"] as? String {
            let bla = 5
            if true {
                let x = 123
            }
            let a = "apple"
        }
    }
```
After a second or two, there is "jump" in the editor (say half a screen), followed by:

![polychromatic_crash](https://cloud.githubusercontent.com/assets/6428065/9174539/c2d1a88a-3f80-11e5-9c34-554cd515c7a7.png)

Methods etc. underneath the "blackened" func also appear black. 

On selecting a piece of the blackened code (e.g. double-clicking in it) a snippet of the code is briefly highlighted correctly, but then goes dark again.

I don't know if this helps you, but Console says:

```
10/08/15 16:14:52,993 Xcode[10184]: [MT] DVTAssertions: Warning in /SourceCache/DVTFrameworks/DVTFrameworks-7714/DVTKit/SourceTextView/TextSystem/DVTSourceTextView.m:5037
Details:  shouldUseTemporaryAttributes failure: curCharRange: {7063, 0}, effRange: {7063, 150}, charIndex=7063
Object:   <DVTSourceTextView: 0x7ff9ed43a010>
Method:   -layoutManager:shouldUseTemporaryAttributes:forDrawingToScreen:atCharacterIndex:effectiveRange:
Thread:   <NSThread: 0x7ff9e9c39710>{number = 1, name = main}
Please file a bug at http://bugreport.apple.com with this warning message and any useful information you can provide.
```

Let me know if you need more feedback.

**Comment by Paul-van-Klaveren** [comment]
Thanks for the quick response! ^_^ I can confirm this fixes the issue Xcode 6.4 (6E35b).
